pkg/utils/secret.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"os"
+)
+
+const (
+	TokenSecretKey     = "token"
+	AccountIdSecretKey = "account-id"
```

Comment:
```suggestion
	AccountIdSecretKey = "account"
```
So we are aligned with the flags we use